packages/sdk/src/utils/chain-constants.ts

```diff
@@ -130,16 +130,16 @@ export const CONTRACT_ADDRESSES: {
   },
   [L2ChainID.OPTIMISM_BEDROCK_LOCAL_DEVNET]: {
     l1: {
-      AddressManager: '0x5FbDB2315678afecb367f032d93F642f64180aa3' as const,
```

Comment:
Perhaps we want to use `getL1ContractsByNetworkName('devnetL1')`

Reply:
I didn't see that in use for the non-Bedrock devnet, so I was unsure if we should convert over to it. Can follow up separately if that's the new best practice.